firebase_functions/functions/src/helpers.ts

```diff
@@ -1,72 +1,63 @@
 import * as crypto from 'crypto';
 const axios = require('axios');
-const { composeAPI, FailMode, LinearWalkStrategy, SuccessMode } = require('@iota/client-load-balancer');
-const { createPrepareTransfers, generateAddress } = require('@iota/core');
-const { asTransactionObject } = require('@iota/transaction-converter');
+import { Client, ClientBuilder } from '@iota/client';
+import { MessageMetadata } from '@iota/client/lib/types';
 const iotaAreaCodes = require('@iota/area-codes');
 const {
   getSettings,
-  updateWalletAddressKeyIndex,
-  updateUserWalletAddressKeyIndex,
   getIotaWallet,
   getUserWallet,
   getGoogleMapsApiKey,
 } = require('./firebase');
 
-const checkRecaptcha = async (captcha, emailSettings) => {
-  const response = await axios({
-    method: 'post',
-    url: `https://www.google.com/recaptcha/api/siteverify?secret=${emailSettings.googleSecretKey}&response=${captcha}`,
-  });
-  return response ? response.data : null;
-};
+/**
+ * Initializes client with devnet node
+ * @returns Client: https://client-lib.docs.iota.org/libraries/nodejs/api_reference.html#clientbuilder
+ */
+const getClient = async (): Promise<Client> => {
+  const settings = await getSettings();
+  const client = new ClientBuilder()
+    .node(settings.provider)
+    .localPow(true)
```

Comment:
I would prefer to run PoW remotely, to reduce load and follow-up cost for the cloud instance.
Could you please check if the selected provider supports remote PoW?